x/market/keeper/keeper_test.go

```diff
@@ -805,9 +805,9 @@ func TestRestrictedDenominations1(t *testing.T) {
 
 	// Restrict trading of gbp
 	k.authorityk = dummyAuthority{
-		RestrictedDenoms: []types2.RestrictedDenom{
+		RestrictedDenoms: types2.RestrictedDenoms{Denoms: []types2.RestrictedDenom{
```

Comment:
A specific alias vs types2 helps to discern the import.

Reply:
Thanks good 👁️ .  🌷